From 20125836458011f8d28851a76e317b9f5a772885 Mon Sep 17 00:00:00 2001
From: Vitaly Wool <vitaly.wool@konsulko.com>
Date: Fri, 17 Nov 2017 17:13:06 +0200
Subject: [PATCH 09/12] Makefile.in: add required libraries

Add Android libraries that are required for connman to be linked
successfully.
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 509347d4..e62ed36e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -131,7 +131,7 @@ src_connmand_SOURCES = $(gdhcp_sources) $(gweb_sources) $(backtrace_sources) \
 
 src_connmand_LDADD = gdbus/libgdbus-internal.la $(builtin_libadd) \
 			@GLIB_LIBS@ @DBUS_LIBS@ @XTABLES_LIBS@ @GNUTLS_LIBS@ \
-			-ldl
+			-ldl -lxtables
 if !ANDROID
 src_connmand_LDADD += -lresolv -lrt
 endif
@@ -299,7 +299,7 @@ client_connmanctl_SOURCES = client/dbus_helpers.h client/dbus_helpers.c \
 			client/main.c
 
 client_connmanctl_LDADD = gdbus/libgdbus-internal.la @DBUS_LIBS@ @GLIB_LIBS@ \
-				-lreadline -ldl
+				-lreadline -ldl -ltermcap
 endif
 
 noinst_PROGRAMS += unit/test-ippool
-- 
2.11.0

